templates/navigation/oe-corporate-blocks-footer.html.twig

```diff
@@ -10,7 +10,111 @@
  * @see templates_preprocess_oe_corporate_blocks_footer()
  */
 #}
-{% include '@ecl/ec-component-footer' with {
-  'corporate_footer': corporate_footer|default([]),
-  'custom_footer': custom_footer|default([]),
+{% set _back_to_top = {
```

Comment:
Question about this implementation: We have a lot of logic in the twig file, shouldn't we do this in the preprocess file?

Reply:
done.